tests/unit/controllers/problemSet.test.ts

```diff
@@ -306,6 +312,36 @@ describe("Problem sets controller tests", () => {
             sinon.assert.calledWith(mockRes.json, updatedUser);
         });
 
+        it("status 200: returns successfully unchanged problemCount", async () => {
+            stubs.validators.validationResult.returns(
+                <any>emptyValidationError()
+            );
+            stubs.problemSetDB.find.resolves(testProblemSetModel1);
+
+            const updatedUser = JSON.parse(JSON.stringify(testProblemSet1));
+            updatedUser._id = req.params.problemSetId;
+            updatedUser.title = "Test Problem Set Title 1 Updated";
+            updatedUser.problemCount = 9;
+            stubs.problemSetDB.update.resolves(updatedUser);
```

Comment:
Confusing variable name `updatedUser`. Change to `updatedProblemSet`.
Also, why do you need to stringify then parse? Isn't `testProblemSet1` already a javascript object (IProblemSet)?

Reply:
Kept both in line with test that precedes - will change after I update with Val's changes.